axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
+
+    temp = player.history[:]
+    player.history = player.use_history[:]
+
```

Comment:
and step 3 is the important one right? Step 3 is the step that's ensuring the state is updated "properly".
--